tests/integration/lsp_tests.rs

```diff
@@ -5076,7 +5076,7 @@ fn lsp_jsr_auto_import_completion() {
     json!({ "triggerKind": 1 }),
   );
   assert!(!list.is_incomplete);
-  assert_eq!(list.items.len(), 261);
+  assert_eq!(list.items.len(), 268);
```

Comment:
Yikes, this is a bit concerning that adding new globals requires changes in seemingly unrelated tests for the LSP. I've opened https://github.com/denoland/deno/issues/23102 to tackle that.